templates/common/gcp/files/etc-kubernetes-credential-providers-gcr-credential-provider.yaml

```diff
@@ -6,9 +6,13 @@ contents:
     kind: CredentialProviderConfig
     providers:
       - name: gcr-credential-provider
+        apiVersion: credentialprovider.kubelet.k8s.io/v1
```

Comment:
Hmm, what are these changes for?

Reply:
weird, I didn't add that. Must have snuck in, will remove.